Lib/http/server.py

```diff
@@ -861,21 +885,37 @@ def translate_path(self, path):
             path += '/'
         return path
 
-    def copyfile(self, source, outputfile):
-        """Copy all data between two file objects.
+    def copyfile(self, source, outputfile, range=None):
```

Comment:
```suggestion
    def copyfile(self, source, outputfile, *, range=None):
```

Now that I have more insight in Python, I think it'd be better if we make it a keyword-only argument. While it shadows the range builtin inside the function, I don't have a better name.

Reply:
I still want this to be a keyword-only argument.

Reply:
I changed that locally but forgot to commit it.😢 Committed now.